docs/table/index.rst

```diff
@@ -99,15 +99,17 @@ You can get summary information about the table as follows::
      c    str1         Column
      d float64 m / s Quantity
 
-From within a Jupyter notebook, the table is displayed as a formatted HTML
-table (details of how it appears can be changed by altering the
-``astropy.table.default_notebook_table_class`` configuration item):
+From within a `Jupyter notebook <https://jupyter.org/>`_, the table is
+displayed as a formatted HTML table (details of how it appears can be changed
+by altering the `astropy.table.conf.default_notebook_table_class
+<astropy.table.Conf.default_notebook_table_class>` item in the
+:ref:`astropy_config`:
```

Comment:
@nstarman, is this what you had in mind?

Reply:
Exactly! Thanks.